api/src/main/java/org/door2door/entities/TeamEntity.java

```diff
@@ -0,0 +1,138 @@
+package org.door2door.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "organization")
```

Comment:
Lets change the db name to match.. no need to keep it organization at this point.